src/inputs/internal/DatePickerOverlay.tsx

```diff
@@ -48,11 +53,13 @@ export function DatePickerOverlay(props: DatePickerOverlayProps) {
         weekdayElement={Weekday}
         selectedDays={[value]}
         initialMonth={value ?? new Date()}
-        onDayClick={(day) => {
+        onDayClick={(day, modifiers) => {
+          if (modifiers.disabled) return;
```

Comment:
Interesting, is this per day?

Reply:
yeah exactly - the callback gets passed the `day` that is clicked, any `modifies` on the actual day (in this case we've set the `disabled` modifier and lastly the `event` object, which we're not using
https://react-day-picker.js.org/api/DayPicker/#onDayClick